GridLayout panics with a nondescriptive error message during rendering if widget.GridLayoutOpts.Columns(...) isn't specified at creation

When a GridLayout is created without specifying `widget.GridLayoutOpts.Columns(...)`, the program will panic.
Unlike the widgets, it doesn't seem like `widget.NewGridLayout(...)` will validate itself. Maybe it should do some validation like how `widget.NewButton(...)` does it?

## Minimum Reproducible Example
```Go
package main

import (
	"github.com/hajimehoshi/ebiten/v2"
	"log"

	"github.com/ebitenui/ebitenui"
	"github.com/ebitenui/ebitenui/widget"
)

type Game struct {
	UI *ebitenui.UI
}

func main() {
	ebiten.SetWindowSize(900, 800)
	ebiten.SetWindowTitle("Ebiten UI Hello World")
	ebiten.SetWindowResizingMode(ebiten.WindowResizingModeEnabled)

	rootContainer := widget.NewContainer()

	eui := &ebitenui.UI{
		Container: rootContainer,
	}

	game := Game{
		UI: eui,
	}

	gridContainer := widget.NewContainer(
		widget.ContainerOpts.Layout(widget.NewGridLayout()),
	)

	rootContainer.AddChild(gridContainer)

	if err := ebiten.RunGame(&game); err != nil {
		log.Print(err)
	}
}

func (g *Game) Update() error {
	g.UI.Update()
	return nil
}

func (g *Game) Draw(screen *ebiten.Image) {
	g.UI.Draw(screen)
}

func (g *Game) Layout(outsideWidth int, outsideHeight int) (int, int) {
	return outsideWidth, outsideHeight
}

```

## Error Log
```Go
panic: runtime error: makeslice: len out of range

goroutine 38 [running]:
github.com/ebitenui/ebitenui/widget.(*GridLayout).preferredColumnWidthsAndRowHeights(0xc0001bcd20, {0x0, 0x0, 0x73d47b?})
	D:/Go/pkg/mod/github.com/ebitenui/ebitenui@v0.5.8/widget/gridlayout.go:210 +0xa5
github.com/ebitenui/ebitenui/widget.(*GridLayout).Layout(0xc0001bcd20, {0x0, 0x0, 0x72a0fd?}, {{0x2187492d628?, 0xc000279a48?}, {0x71f27f?, 0x9f0460?}})
	D:/Go/pkg/mod/github.com/ebitenui/ebitenui@v0.5.8/widget/gridlayout.go:112 +0x66
github.com/ebitenui/ebitenui/widget.(*Container).doLayout(...)
	D:/Go/pkg/mod/github.com/ebitenui/ebitenui@v0.5.8/widget/container.go:243
github.com/ebitenui/ebitenui/widget.(*Container).Render(0xc0001bcd90, 0xc0002bc070, 0xa64d08)
	D:/Go/pkg/mod/github.com/ebitenui/ebitenui@v0.5.8/widget/container.go:227 +0xc3
github.com/ebitenui/ebitenui/widget.(*Container).Render(0xc0001bccb0, 0xc0002bc070, 0xa64d08)
	D:/Go/pkg/mod/github.com/ebitenui/ebitenui@v0.5.8/widget/container.go:236 +0x19d
github.com/ebitenui/ebitenui/widget.renderDeferredRenderQueue(0xc0002bc070)
	D:/Go/pkg/mod/github.com/ebitenui/ebitenui@v0.5.8/widget/widget.go:688 +0x95
github.com/ebitenui/ebitenui/widget.RenderWithDeferred(...)
	D:/Go/pkg/mod/github.com/ebitenui/ebitenui@v0.5.8/widget/widget.go:676
github.com/ebitenui/ebitenui.(*UI).render(0xc0001d8480, 0xc0002bc070)
	D:/Go/pkg/mod/github.com/ebitenui/ebitenui@v0.5.8/ui.go:254 +0x470
github.com/ebitenui/ebitenui.(*UI).Draw(0xc0001d8480, 0xc0002bc070)
	D:/Go/pkg/mod/github.com/ebitenui/ebitenui@v0.5.8/ui.go:58 +0xee
main.(*Game).Draw(0xc000279d20?, 0x976f8a?)
	D:/Sources/ebitenui-dvz-mre/main.go:47 +0x16
github.com/hajimehoshi/ebiten/v2.(*gameForUI).DrawOffscreen(0xc0001bce00)
	D:/Go/pkg/mod/github.com/hajimehoshi/ebiten/v2@v2.6.6/gameforui.go:152 +0x27
github.com/hajimehoshi/ebiten/v2/internal/ui.(*context).drawGame(0xc0001d8f80, {0xabab00, 0xc0000d9b80}, 0x0)
	D:/Go/pkg/mod/github.com/hajimehoshi/ebiten/v2@v2.6.6/internal/ui/context.go:194 +0x11d
github.com/hajimehoshi/ebiten/v2/internal/ui.(*context).updateFrameImpl(0xc0001d8f80, {0xabab00, 0xc0000d9b80}, 0x1, 0x408c200000000000, 0x4089000000000000, 0x3ff0000000000000, 0xd33100, 0x0, 0xc0000220c0)
	D:/Go/pkg/mod/github.com/hajimehoshi/ebiten/v2@v2.6.6/internal/ui/context.go:162 +0x4cf
github.com/hajimehoshi/ebiten/v2/internal/ui.(*context).updateFrame(0xc0001d8f80, {0xabab00, 0xc0000d9b80}, 0x408c200000000000, 0x4089000000000000, 0x3ff0000000000000, 0xd33100, 0xc0000220c0)
	D:/Go/pkg/mod/github.com/hajimehoshi/ebiten/v2@v2.6.6/internal/ui/context.go:75 +0x90
github.com/hajimehoshi/ebiten/v2/internal/ui.(*userInterfaceImpl).updateGame(0xd33100)
	D:/Go/pkg/mod/github.com/hajimehoshi/ebiten/v2@v2.6.6/internal/ui/ui_glfw.go:1231 +0x18a
github.com/hajimehoshi/ebiten/v2/internal/ui.(*userInterfaceImpl).loopGame(0xd33100)
	D:/Go/pkg/mod/github.com/hajimehoshi/ebiten/v2@v2.6.6/internal/ui/ui_glfw.go:1199 +0xac
github.com/hajimehoshi/ebiten/v2/internal/ui.(*userInterfaceImpl).Run.func2()
	D:/Go/pkg/mod/github.com/hajimehoshi/ebiten/v2@v2.6.6/internal/ui/run_glfw_notsinglethread.go:60 +0x37
golang.org/x/sync/errgroup.(*Group).Go.func1()
	D:/Go/pkg/mod/golang.org/x/sync@v0.6.0/errgroup/errgroup.go:78 +0x56
created by golang.org/x/sync/errgroup.(*Group).Go in goroutine 1
	D:/Go/pkg/mod/golang.org/x/sync@v0.6.0/errgroup/errgroup.go:75 +0x96
```

--- a/widget/gridlayout.go
+++ b/widget/gridlayout.go
@@ -63,7 +63,15 @@ func NewGridLayout(opts ...GridLayoutOpt) *GridLayout {
 		o(g)
 	}
 
+	g.validate()
+
 	return g
+}
+
+func (gl *GridLayout) validate() {
+	if gl.columns == 0 {
+		panic("GridLayout: columns is required.")
+	}
 }
 
 // Columns configures a grid layout to use c columns.

